docs/docs/workflows/permissions.md

```diff
@@ -0,0 +1,45 @@
+---
+id: permissions
+title: Permissions
+---
+
+Permissions in **ToolJet Workflows** provide a structured approach to access control, ensuring precise management of who can view, edit, or execute workflows. The below table gives a detailed summary of permissions in context of ToolJet Workflows. 
```

Comment:
Should we mention **beta** here?

Reply:
```suggestion
Permissions in **ToolJet Workflows (Beta)** provide a structured approach to access control, ensuring precise management of who can view, edit, or execute workflows. The below table gives a detailed summary of permissions in context of ToolJet Workflows. 
```

Reply:
"Permissions in **ToolJet Workflows (Beta)** provide a structured...."

If we mention Beta like this, it would mean that the mentioned features related to Permissions are working in the beta version which is not the case. I can add the Beta version banner message like I've done in overview if needed.